### DISCUSSION TITLE
Require DCO sign-off on web based commits

### DISCUSSION THREAD

**Discussion by florian-h05** [openhab/openhab-distro#1531] [openHAB Maintainers Discussions]
Hello @openhab/maintainers,

GitHub added a new feature to require DCO sign-off on web based commits some time ago.
Even though most contributions to openHAB aren't web-based, I think it would make sense to enable the sign-off requirement for web based commits to avoid trouble with the DCO check.

This can be enabled per repo or organization wide @kaikreuzer, see https://github.blog/changelog/2022-06-08-admins-can-require-sign-off-on-web-based-commits/.

Best regards,
Florian

**Comment by kaikreuzer** [comment]
Thanks @florian-h05, I didn't know this setting yet - I have enabled it for all repos as suggested!

**Reply by Confectrician** [reply]
All repos includes the docs?
We agreed to skip dco for the docs and aside the docs repo will propbably be the one with the most web based commits due to the edit links in the docs.

**Reply by kaikreuzer** [reply]
Why should we skip docs if this feature adds a sign-off for web commits automatically?

**Reply by Confectrician** [reply]
>  to require DCO sign-off

My comment was based on the above quote without further checking the url and that there will be added a sign off statement automatically.
Sorry for disturbing here.
